<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Them_Sinh_Vien</class>
 <widget class="QDialog" name="Them_Sinh_Vien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="Lop">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 2px solid gray;
	border-radius: 20px;
	background-color: #F0F0F0;
	color: black;
	font-size: 16px;
text-align: center;
}

QComboBox::drop-down {
width: 30px;
subcontrol-origin: padding;
subcontrol-position: top right;
border: 2px solid gray;
border-radius: 10px;
background-color: #F0F0F0;
}

QComboBox::down-arrow {
    image: url(:/logo/down-arrow (1).png); /* Đường dẫn hình ảnh mũi tên */
    width: 20px;
    height: 20px;
}

QComboBox QAbstractItemView {
    border: 2px solid gray;
    border-radius: 10px;
    background-color: white;
    selection-background-color: lightblue;  /* Màu nền của mục được chọn */
    selection-color: black;  /* Màu chữ của mục được chọn */
    padding: 5px;
	font-size: 12px;
}

</string>
   </property>
   <property name="placeholderText">
    <string>                  Chọn Lớp</string>
   </property>
  </widget>
  <widget class="QLabel" name="ThongTin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; color:#ff0000;&quot;&gt;Thêm Sinh Viên&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MSSV">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: #F0F0F0;
color: black;
font-size: 16px;
padding-left: 85px;</string>
   </property>
   <property name="placeholderText">
    <string>    MSSV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Ho">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: #F0F0F0;
color: black;
font-size: 16px;
padding-left: 85px;</string>
   </property>
   <property name="placeholderText">
    <string>    Họ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Ten">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 20px;
background-color: #F0F0F0;
color: black;
font-size: 16px;
padding-left: 85px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>    Tên</string>
   </property>
  </widget>
  <widget class="QComboBox" name="GioiTinh">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 2px solid gray;
	border-radius: 20px;
	background-color: #F0F0F0;
	color: black;
	font-size: 16px;
text-align: center;
}

QComboBox::drop-down {
width: 30px;
subcontrol-origin: padding;
subcontrol-position: top right;
border: 2px solid gray;
border-radius: 10px;
background-color: #F0F0F0;
}

QComboBox::down-arrow {
    image: url(:/logo/down-arrow (1).png); /* Đường dẫn hình ảnh mũi tên */
    width: 20px;
    height: 20px;
}

QComboBox QAbstractItemView {
    border: 2px solid gray;
    border-radius: 10px;
    background-color: white;
    selection-background-color: lightblue;  /* Màu nền của mục được chọn */
    selection-color: black;  /* Màu chữ của mục được chọn */
    padding: 5px;
	font-size: 12px;
}

</string>
   </property>
   <property name="placeholderText">
    <string>                      Giới Tính</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoiLop">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LoiMSSV">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LoiTen">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LoiHo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LoiGioiTinh">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Them_Sinh_Vien</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Them_Sinh_Vien</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
